.github/CODEOWNERS

```diff
@@ -26,6 +26,7 @@ yarn.lock                                           @backstage/community-plugins
 /workspaces/github-pull-requests-board              @backstage/community-plugins-maintainers  @gregorytalita
 /workspaces/kafka                                   @backstage/community-plugins-maintainers  @andrewthauer
 /workspaces/linguist                                @backstage/community-plugins-maintainers  @awanlin
+/workspaces/nexus-repository-manager                @backstage/community-plugins-maintainers  @jessicajhee
```

Comment:
Also for codeowners, we'd want to add someone who is familiar with the plugin. Maybe @schultzp2020 since he made [the initial plugin PR](https://github.com/janus-idp/backstage-plugins/pull/618)?

Reply:
Okay! I've change it to Paul for now - @schultzp2020 what do you think?